src/api.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { initHappychat, eventAPI } from './index';
+
+const api = {
+	/**
+	 * Initialize/Open happychat client. This method accepts an {Object} parameter with the 
```

Comment:
Isn't this already documented in https://github.com/Automattic/happychat-client/blob/76b85664f4dcdc2c7a98b6d34e4c017278b70577/src/index.js#L234? I don't see any value in duplicating the `jsdocs` and worry that it becomes a burden to update two places and, eventually, both become obsolete.

Reply:
I copied the documentation here because I'm currently refactoring the `initHappychat` method because it's too tight to WPCOM token authentication and we need it to work with other authentication strategies for calypso. And I think it would be better to have docs at the api's entry point.

Reply:
If we plan on doing that: 

- we should remove the jsdocs at `src/index`.
- perhaps take this opportunity to document the rest of the methods

Reply:
> we should remove the jsdocs at src/index.

I removed it

> perhaps take this opportunity to document the rest of the methods

I haven't worked on that part yet and the `eventAPI` has no documentation at all which makes it pretty difficult for me to document it, especially giving the fact that I don't have that much bandwidth at the moment.

Reply:
Added initial docs for the `eventsAPI` in 036a38b36e05cb4244f2d3c18548589f1ffacb11 I committed directly to the branch to avoid back-and-forths, I hope you don't mind and it doesn't disrupt your workflow.

Note that the events API is something I wrote for integrating Happychat v1 in WooCommerce. Happychat v2 doesn't need them and I was hoping to remove that part of the code (more reductions to bundle size!). I'm not sure if that will be useful for Calypso integration, but in case it's not, please, do not hesitate in delete it.

Reply:
Thanks for doing that 🏅